spec/core/configuration_spec.rb

```diff
@@ -21,6 +21,7 @@
       allow(connector_settings).to receive(:needs_service_type?).and_return(needs_service_type)
       allow(connector_settings).to receive(:formatted).and_return('')
       allow(connector_class).to receive(:configurable_fields).and_return(configuration)
+      allow(connector_class).to receive(:configurable_fields_indifferent_access).and_return(configuration.with_indifferent_access)
```

Comment:
Another place with potentially missing `require` statement of  'active_support/core_ext/hash/indifferent_access'